console/console-init/ui/src/pages/AddressSpaceDetail/AddressList/AddressListFilterPage.tsx

```diff
@@ -65,7 +66,9 @@ export const AddressListFilterPage: React.FunctionComponent<AddressListFilterPro
   isPurgeAllDisabled
 }) => {
   const { name, namespace, type } = useParams();
-  const [addressSpacePlan, setAddressSpacePlan] = React.useState();
+  const [addressSpacePlan, setAddressSpacePlan] = React.useState<string | null>(
```

Comment:
why do we need null type with string. some times null and undefined cause the issue . avoid to use null and undefined as much as possible. in case you want to set state empty just use blank string i.e. "".

Reply:
server doesn't support empty string, we have pass the value as null. This is main fix for this PR.